src/Form/CertificationRequestType.php

```diff
@@ -4,16 +4,22 @@
 
 use App\Entity\CertificationRequest;
 use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
 use Symfony\Component\Form\Extension\Core\Type\FileType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints as Assert;
 
 class CertificationRequestType extends AbstractType
 {
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
             ->add('document', FileType::class)
+            ->add('cgu', CheckboxType::class, [
+                'mapped' => false,
+                'constraints' => [new Assert\IsTrue()],
```

Comment:
same comment as the other PR. The error mesage if not checked can be improved IMHO

Reply:
rebased on `checkbox-constraints` and added translation validation message. ty